app/models/Comment.java

```diff
@@ -32,4 +32,9 @@ public Comment(Post post, String author, String content) {
         this.postedAt = new Date();
     }
 
+    public static List<Comment> findCommentedOnBy(String profile) {
```

Comment:
We need to start to think about variable management. Profile means different things in different places of this project. I think here you are actually passing the profileName or userName right?